nucypher/acumen/nicknames.py

```diff
@@ -22,41 +22,91 @@
 
 import unicodedata
 
-HERE = BASE_DIR = abspath(dirname(__file__))
+HERE = abspath(dirname(__file__))
 with open(join(HERE, 'web_colors.json')) as f:
     colors = json.load(f)
 
 colors = colors['colors']
-pairs = []
-
-symbols_tuple = ("♈", "♉", "♊", "♋", "♌", "♍", "♎", "♏", "♐", "♑", "♒", "♓",
-                 "♚", "♛", "♜", "♝", "♞", "♟", "⚓", "⚔", "⚖", "⚗", "⚑", "⚘",
-                 "⚪", "⚵", "⚿", "⛇", "⛈", "⛰", "⛸", "⛴", "⛨", "✈", "☤",
-                 "⏚", "☠", "☸", "☿", "☾", "♁", "♃", "♄", "☄", "☘", "⚜", "⚚",
-                 "⏲", "☣", "☥", "♣", "♥", "♦", "♠", "♫", "🟒", "⚛", "⚙", "⎈",
-                 "☮", "☕", "☈", "♯", "♭")
-
-
-def nicename(symbol):
-    unicode_name = unicodedata.name(symbol)
-    final_word = unicode_name.split()[-1]
-    if final_word in ("SYMBOL", "SUIT", "SIGN"):
-        final_word = unicode_name.split()[-2]
-    return final_word.capitalize()
-
-
-def nickname_from_seed(seed, number_of_pairs=2):
-    # TODO: #1823 - Workaround for new nickname every restart
-    # if not seed:
-    #     raise ValueError("No checksum provided to derive nickname.")
-    symbols = list(symbols_tuple)
-
-    random.seed(seed)
-    pairs = []
-    for pair in range(number_of_pairs):
-        color = random.choice(colors)
-        symbol = random.choice(symbols)
-        symbols.remove(symbol)
-        pairs.append((color, symbol))
-    nickname = " ".join(("{} {}".format(c['color'], nicename(s)) for c, s in pairs))
-    return nickname, pairs
+
+symbols = {
```

Comment:
Shouldn't this be `SYMBOLS`?

Reply:
Or `_SYMBOLS`, even.

Reply:
I suppose so, and the other constants as well.

Reply:
Fixed.